remote-reader/app/src/main/res/xml/settings.xml

```diff
@@ -52,7 +58,7 @@
 
         <ListPreference
             android:key="timeout"
-            android:defaultValue="500"
+            android:defaultValue="10000"
```

Comment:
I think it would be better to keep the smaller default value, because the I/O is done on the main thread. Waiting 10s for a card would block the UI for this time.

The bigger waiting time would be great, if the I/O operations would be done in a worker thread. Unfortunately, I never found time to implement this...

Reply:
True, thanks for suggestion, I will revert this back